model.go

```diff
@@ -96,17 +122,15 @@ type Persistable interface {
 // Writable must be implemented by those values that defines internally
 // if they can be sent back to the database to be stored with its changes.
 type Writable interface {
-	// IsWritable returns whether this Model can be sent back to the database
-	// to be stored with its changes.
+	// IsWritable returns whether this Model can be saved into the database.
 	IsWritable() bool
 	setWritable(bool)
 }
 
-// ColumnAddresser must be implemented by those values that expose their properties
-// under pointers, identified by its property names
+// ColumnAddresser provides the pointer addresses of columns.
```

Comment:
If this interface is not needed outside, it should be maybe made private.
If must be public for internal use, it should be explained in the doc;

Please, take this in consideration for the rest of the project :)